assets/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;400;500;600&display=swap" rel="stylesheet">
+    <title>Interactive Rating Component</title>
+</head>
+<body>
+    <!-- Section to hold the cards -->
+    <section class="rate-page">
+        <!-- Rate Card Start -->
+        <div class="rate-card">
+            <div class="circle">
+                <div class="icon-star">
+                    <img src="./img/icon-star.svg" alt="">
+                </div>
+            </div>
+            
+            
+            <div class="card-heading">
+                <h1>How did we do?</h1>
+            </div>
+            <div class="card-text">
+                <p>Please let us know how we did with your support request. All feedback is appreciated to help us improve in our offering!</p>
+            </div>
+            <div class="card-button">
+                <button class="rate-point">1</button>
+                <button class="rate-point">2</button>
+                <button class="rate-point">3</button>
+                <button class="rate-point">4</button>
+                <button class="rate-point">5</button>
```

Comment:
So, because these are meant to be interactive elements, they need to be input elements. Buttons may seem like a good option too but they are for actions and not interactions. We will also need to collect the value of the rate that was clicked, and buttons do not store that. A better input element would be the [radio button ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio)then style it to look like the designs.